controllers/csm_controller_test.go

```diff
@@ -394,7 +414,7 @@ func (suite *CSMControllerTestSuite) TestCsmPreCheckVersionError() {
 func (suite *CSMControllerTestSuite) TestCsmPreCheckTypeError() {
 
 	csm := shared.MakeCSM(csmName, suite.namespace, configVersion)
-	csm.Spec.Driver.CSIDriverType = csmv1.PowerFlex
+	csm.Spec.Driver.CSIDriverType = csmv1.PowerStore
```

Comment:
Why is the drivertype "Powerstore" when we have not added support for Powerstore yet

Reply:
this is a negative test case 
so we need to provide different driver name and the one supported in csm-operator